src/i18n/en.js

```diff
@@ -672,6 +673,13 @@ const translations = {
   'mobilityPlatform.content.speedLimitZones.area': 'Speed limit zone',
   'mobilityPlatform.content.speedLimitZones.limit': 'Speed limit',
   'mobilityPlatform.content.speedLimitZones.suffix': 'km/h',
+  'mobilityPlatform.content.publicToilets.title': 'Public toilet',
+  'mobilityPlatform.content.publicToilets.openNormalTitle': 'Opening hours:', // TODO verify
+  'mobilityPlatform.content.publicToilets.openSummerTitle': 'Special opening hours:', // TODO verify
+  'mobilityPlatform.content.publicToilets.openNormalDate': 'Between: 1.10 - 30.4',
```

Comment:
Is this information available from the data provided for public WCs? When these change, it will require manual updating at the code, so please tell me it is available dynamically